js/error.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+window.error = (function () {
+  var errorTemplate = document.querySelector('#error').content.querySelector('.error');
+  var errorMessageElement = errorTemplate.cloneNode(true);
+  var errorButton = errorMessageElement.querySelector('.error__button');
+  var errorText = errorMessageElement.querySelector('.error__message');
+
+  return {
+    errorHandler: function (error, errBtnHandler) {
+      errorText.innerText = 'Sending error: ' + error;
+      errorButton.addEventListener('click', errBtnHandler);
+      document.querySelector('body').appendChild(errorMessageElement);
+      var errorElement = document.querySelector('body').querySelector('.error');
+
+      setTimeout(function () {
+        var errorButtonElement = document.querySelector('.error__button');
```

Comment:
using document is not optimal here neither use `errorMessageElement.querySelector('.error__button')`

Reply:
done